app/Models/Feed.php

```diff
@@ -410,7 +410,8 @@ public function loadEntries($simplePie) {
 						$height = $enclosure->get_height();
 						$width = $enclosure->get_width();
 						$length = $enclosure->get_length();
-						if ($medium === 'image' || strpos($mime, 'image') === 0 || ($mime == '' && $length == null && ($width != 0 || $height != 0))) {
+						if ($medium === 'image' || strpos($mime, 'image') === 0 ||
+							($mime == '' && $length == null && ($width != 0 || $height != 0 || preg_match('/[.](gif|jpe?g|png|svg)$/i', $elink)))) {
```

Comment:
Should probably also add WebP and that new one based on AV1.

Reply:
Done https://github.com/FreshRSS/FreshRSS/pull/3861/commits/5fd913d0c8c56638b2998b1359bac4a0e676d620